source/educators/how-tos/set_up_course/add_metadata.rst

```diff
@@ -4,20 +4,22 @@
 Add Course Metadata
 ###################
 
-   .. tags:: educator, how-to
+.. tags:: educator, how-to
 
-  To make certain information about your course available to entities such as
-  customer relationship management (CRM) software, a marketing site, or other
-  external systems, follow these steps.
+To make certain information about your course available to entities such as
+customer relationship management (CRM) software, a marketing site, or other
+external systems, follow these steps.
 
-  #. Create a JSON dictionary that contains the metadata that you want to add.
-  #. In Studio, open your course, and then select **Advanced Settings** on the
-     **Settings** menu.
-  #. In the **Other Course Settings** field, paste your JSON dictionary.
+#. Create a JSON dictionary that contains the metadata that you want to add.
+#. In Studio, open your course, and then select **Advanced Settings** on the
+   **Settings** menu.
+#. In the **Other Course Settings** field, paste your JSON dictionary.
+#. Select **Save Changes**.
+
+In case you can't find the **Other Course Settings** field in
+the **Advanced Settings**, set ``ENABLE_OTHER_COURSE_SETTINGS`` to ``true``
+under ``FEATURES`` in ``/edx/etc/studio.yml`` and restart Studio...
```

Comment:
This is for site operators, right? Might be best to put this in a note directive and say something like, "ask your site operator to set ..."